src/kernel/include/shared/cos_types.h

```diff
@@ -252,10 +252,10 @@ enum
 	BOOT_CAPTBL_COMP0_COMP         = 24,
 	BOOT_CAPTBL_SINV_CAP           = 28,
 	BOOT_CAPTBL_SELF_INITTHD_BASE  = 32,
-	BOOT_CAPTBL_SELF_INITTCAP_BASE = BOOT_CAPTBL_SELF_INITTHD_BASE + NUM_CPU_COS * CAP16B_IDSZ,
-	BOOT_CAPTBL_SELF_INITRCV_BASE  = round_up_to_pow2(BOOT_CAPTBL_SELF_INITTCAP_BASE + NUM_CPU_COS * CAP16B_IDSZ,
+	BOOT_CAPTBL_SELF_INITTCAP_BASE = BOOT_CAPTBL_SELF_INITTHD_BASE + (NUM_CPU_COS + 1) * CAP16B_IDSZ,
```

Comment:
You should use `NUM_CPU` here. Because `NUM_CPU_COS` wastes a capability slot when we work on single core, that is when NUM_CPU is 1.

Reply:
I will use NUM_CPU instead.

Reply:
I don't know why we have `NUM_CPU_COS`, it's from old code/kernel I think.